pages/labTwo.html

```diff
@@ -1,39 +1,170 @@
+                    <td> per minute</td>
+                </tr>
+            </table>
+        </div>
+        <div>
+            <label>Patient Blood Pressure</label>
+            <table>
+                <tr>
+                    <td><label> Systolic Pressure</label>
+                    <td><input id="syspressure" type="number" required>
+                    <td>mm Hg</td>
+
+                </tr>
+                <tr>
+                    <td><label> Diastolic Pressure</label>
+                    <td><input id="diaspressure" type="number" required>
+                    <td>mm Hg</td>
+                </tr>
+            </table>
+        </div>
+        <div>
+            <table>
+                <label>Respiratory rate</label>
+                <tr>
+                    <td><input id="resprate" name="resprate" type="number" min="1" placeholder="Eg. 12" required></td>
 
-    <!-- ################################################################################################ -->
-    <p>Copyright &copy; 2016 - Sharmistha Dutta & Andrew Brown - All Rights Reserved -</p>
+                    <td>breaths per minute</td>
+                </tr>
+            </table>
+
+        </div>
+    </fieldset>
+
+    <fieldset>
+        <legend>Universal Pain Assessment Tool</legend>
+        <div>
+            <label>Pain
+                <input id="pain" name="pain" list="pain-level" type="text" required>
+                <datalist id="pain-level">
+                    <option label="no pain" value="0"></option>
+                    <option label="mild pain" value="1-3"></option>
+                    <option label="moderate pain" value="3-7"></option>
+                    <option label="Severe pain" value="8-9"></option>
+                    <option label="Worst pain possible" value="10"></option>
+                </datalist>
+            </label>
+        </div>
+    </fieldset>
+
+    <fieldset>
+        <legend>Pulse oximiter measurement</legend>
+        <div>
+            <table>
+                <label>Oxygen saturation </label>
+                <tr>
+                    <td><input id="oxygensaturation" name="oxygensaturation" type="number"
+                               placeholder="Enter a percentage" required></td>
+                    <td> %</td>
+                </tr>
+            </table>
+        </div>
+    </fieldset>
+
+    <fieldset>
+        <legend>Blood glucose level</legend>
+        <table>
+            <tr>
+                <td><input id="bloodglucoselevel" name="bloodglucoselevel" type="number" step="0.1"
+                           placeholder="Enter a bloodglucoselevel" required></td>
+                <td>mmol/L</td>
+            </tr>
+        </table>
+    </fieldset>
+
+    <div>
+        <button type="button" onclick="myFunction()">Submit</button>
+    </div>
+</form>
+
+<script>
+    // to display the date and time using javascript
+    document.getElementById("datenTime").value = Date();
```

Comment:
You have jQuery included in the project... why don't you use it? Additionally, there may be some jQuery helper functions (or maybe in a plugin) to validate the responses inline instead of as alert boxes. Ideally, if validation fails, the user will see red failure messages at the places he/she must fix in the form